scene/2d/tile_map_layer.cpp

```diff
@@ -301,6 +304,9 @@ void TileMapLayer::_rendering_update(bool p_force_cleanup) {
 					if (prev_ci == RID() || prev_material != mat || prev_z_index != tile_z_index) {
 						// If so, create a new CanvasItem.
 						ci = rs->canvas_item_create();
+						if (!is_physics_interpolated_and_enabled()) {
```

Comment:
Thinking about it, this might be slightly more efficient when interpolation is off globally, i.e. we only need to turn it off in `VisualServer` if global interpolation is on (as this case is dealt with already server side, when globally off).
```
if (is_inside_tree() && get_tree()->is_physics_interpolation_enabled() && !is_physics_interpolated()) {
```
The flip side is that you would have to handle switching physics interpolation on globally on the fly (I don't know if we actually properly handle that at the moment, but if not it would just propagating a change throughout the tree I think).